### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by byteconde**
What does this feature do?
- If user exists, display message ('User recognized')
- If user doesn't exists, then push into users  temporal **fake** database
- Store user db in local storage

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -12,7 +12,9 @@
 
 <body>
   <h1>Tutopia</h1>
+
   <script src="js/main.js"></script>
+  <script src="js/script.js"></script>
```
Comment (DiegoCuevas) [inline public/index.html]: why are you using two scripts?

File: public/js/script.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+function signIn(user) {
+  if(!compareName(user)) {
+    users.push({
+      id: new Date().getTime(),
+      username: user,
+      channels: [],
+      messages: []
+    })
+  } else {
+    alert("Usuario reconocido")
```
Comment (christopher77) [inline public/js/script.js]: I think it's better to continue coding in english even in the messages

File: public/index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (byteconde) [reply public/index.html]: @DiegoCuevas  sorry I was testing, I'll send a new PR.

File: public/js/script.js (hunk 2 above)
Comment (byteconde) [reply public/js/script.js]: Fixed, thanks !

File: public/js/script.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+const channels = [
+  {
+    id: 1,
+    messages: [1,2],
+    name: "general"
+  }
+]
+
+let userNamePrompt = prompt("What's your username?");
```
Comment (condef5) [inline public/js/script.js]: The username should be requested only once.
if I open another tab you should not ask me for my username

**Pull request closed without merging**